app/src/test/kotlin/fr/free/nrw/commons/explore/depictions/WikidataItemDetailsActivityUnitTests.kt

```diff
@@ -83,17 +77,8 @@ class WikidataItemDetailsActivityUnitTests {
         )
         Whitebox.setInternalState(activity, "supportFragmentManager", supportFragmentManager)
 
-        parent =
-            LayoutInflater.from(activity).inflate(R.layout.activity_wikidata_item_details, null)
-
-        mediaContainer = parent.findViewById(R.id.mediaContainer)
-        Whitebox.setInternalState(activity, "mediaContainer", mediaContainer)
-
-        tabLayout = parent.findViewById(R.id.tab_layout)
-        Whitebox.setInternalState(activity, "tabLayout", tabLayout)
+        binding = ActivityWikidataItemDetailsBinding.inflate(LayoutInflater.from(activity))
```

Comment:
If the rest of the test isnt using this instance of the binding, perhaps delete the reference?